10-JavaScript/fun-with-functions.md

```diff
@@ -15,12 +15,12 @@ All functions in JavaScript are variables. Just like any other variable, you dec
 How do you set the value to be a function? With the `function` keyword of course!
 
 ```javascript
-const bark = function() {
-  console.log("Woof!");
+const bark = function bark() {
+  console.log('Woof!');
 };
 ```
 
-**Note** we declared the function `bark` here with `const`.  We could have used `let` or `var`, but because we don't want `bark` to be reassigned, it makes sense to make it a constant.
+**Note** we declared the function `bark` here with `const`.  We could have used `let` or `var`, but because we don't want `bark` to be reassigned, it makes sense to make it a constant.  We also added the function name to the right of the `function` keyword.  This is optional, but helps identify things in the stack trace.
```

Comment:
I think the stack trace should be fine either way. AFAIA, naming your function variables is only useful if you want to use recursion:

https://stackoverflow.com/questions/34917414/var-name-function-name-function-name-used-twice